config/general.yml-example

```diff
@@ -280,6 +280,17 @@ THEME_BRANCH: false
 # ---
 FORCE_REGISTRATION_ON_NEW_REQUEST: false
 
+# Does a user needs to pass a RECAPTCHA test before being allowed to
```

Comment:
Oh, hang on - I pinched that from the line above `FORCE_REGISTRATION_ON_NEW_REQUEST` fixing both while I'm here